src/core.ts

```diff
@@ -31,14 +31,14 @@ export async function lint(
   const program = ts.createProgram(rootNames, compilerOptions, undefined, oldProgram)
   const checker = program.getTypeChecker()
 
-  const allFiles: string[] = []
+  const allFiles = {} as {[file: string]: boolean}
```

Comment:
I think it's better to use `const allFiles = new Set<string>()` here

Reply:
Agree, do it yet